src/main/kotlin/org/ageseries/libage/api/Server.kt

```diff
@@ -0,0 +1,56 @@
+package org.ageseries.libage.api
+
+import com.google.gson.Gson
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import org.ageseries.libage.sim.electrical.mna.Circuit
+import org.ageseries.libage.sim.electrical.mna.component.Component
+import org.ageseries.libage.sim.electrical.mna.component.Resistor
+import org.ageseries.libage.sim.electrical.mna.component.VoltageSource
+import java.util.*
+import kotlin.collections.MutableList
+import kotlin.collections.mutableListOf
+import kotlin.collections.mutableMapOf
+import kotlin.collections.set
+
+data class CircuitBundle(var circuit: Circuit, var components: MutableList<Component>)
+
+val circuits = mutableMapOf<UUID, CircuitBundle>()
+
+fun loadTestCircuits() {
```

Comment:
This is just the first test from the MNATests.kt file. It doesn't need to be added.